jekyll.gemspec

```diff
@@ -36,4 +36,5 @@ Gem::Specification.new do |s|
   s.add_runtime_dependency('rouge', '~> 1.7')
   s.add_runtime_dependency('jekyll-sass-converter', '~> 1.0')
   s.add_runtime_dependency('jekyll-watch', '~> 1.1')
+  s.add_runtime_dependency("pathutil", "~> 0.9")
```

Comment:
@ptoomey3 yes, we don't respond to some methods from Pathname like cleanpath (which Jekyll doesn't use) because we do all that internally where we can or let StdLib deal with most of it, but I'm happy to implement it if you like.  For the most part, 90% of the methods that we have on that class are delegated, with the exception of a few:

- `#read` diverges, it normalizes and encodes by default (this was designed for Jekyll.)
- `#write` diverges, it normalizes and encoded by default (this was designed for Jekyll.)
- We offer `read_yaml` and `read_json` both written with my own library in mind and Jekyll.  Right now Jekyll relies heavily on SafeYAML, while YAML itself offers Yaml#safe so this wraps around both and looks for the latter while falling back to the former.
- We offer >, <, >=, <=, =~ and others which check if it's below, above, or like and also allows for a Regexp match of the said, sometimes people want Regexp's, so we made it available.
- Our glob is "safer", in that it will `chdir` before doing the glob to avoid any potential issues with globbing, this was a bug in Jekyll.
- Some methods that Pathname had customized, we route to their real classes directly (those can all be found at the bottom of the class, this is where most of the methods come from, we don't directly do them, we actually delegate them out.)

On the `#read`, `#write` when a read comes in and it's Windows, we actually convert to Linux and then if it's Windows we convert back to Windows.  We enforce UTF-8 by default, which should solve a lot of Jekyll problems, and we also allow the default encoding to be set so Jekyll's encoding can not be Pathutils.  That specific part was designed explicitly for Jekyll and falls along the lines of what I had mentioned in the past "lets normalize everything and then normalize it back out."